.github/workflows/label-pr-config.yml

```diff
@@ -0,0 +1,195 @@
+## Order of entries in this map *does* matter for the resolved labels
```

Comment:
Doesn't GitHub try to interpret this file as a workflow?

Reply:
Oh, you're probably onto something, yes! And checking my email today, I found this job error which confirms your suspicions: [phillipj/node action run](https://github.com/phillipj/node/actions/runs/764994211).

Compared to the other repos where I've tested this so far, I've put this config file in `./github`. But here in nodejs/node it felt slightly more natural for that file to not live alongside the .md files found in `./github`.

I did not consider that fact that it's a .yml file like another workflow file.

I'll move it `./github/workflows -> ./github`.

Reply:
> 
> I'll move it `./github/workflows -> ./github`.

Other than clutter, is there any reason not to put the config in the root (`./`)?

Reply:
Nope, it could be located anywhere in the repository. Even a "hidden" file with `.`-prefix would work I assume.

I'm not opinionated about its location to be honest. If several of you would rather see this in `./` or similar, I'd happily move it & update the Action configuration again, just shout 😄